### PR TITLE
An Inspector Doesn't Call: Deleting the server side support for the Emacs Inspector

### PR DESCRIPTION
**Pull request by VC1995**
picked up from #An Inspector Doesn't Call #1795

### CODE REVIEW

**Comment by fommil** [comment]
you can also delete all the methods in the RichPresentationCompiler that are no longer used.

**Comment by VC1995** [comment]
Okay :+1:

File: core/src/main/scala/org/ensime/core/Completion.scala [hunk 1]
```diff
@@ -39,19 +39,18 @@
...
-
 import akka.actor.ActorRef
 import akka.pattern.Patterns
 import akka.util.Timeout
 import org.ensime.api._
-import org.ensime.indexer.PackageName
 import org.ensime.indexer.lucene.SimpleLucene
 import org.ensime.util.Timing.dilation
 
+import scala.collection.mutable
```
Comment (fommil) [inline core/src/main/scala/org/ensime/core/Completion.scala]: import weirdness

File: core/src/main/scala/org/ensime/core/RichPresentationCompiler.scala [hunk 2]
```diff
@@ -38,13 +38,22 @@
...
+import org.ensime.indexer._
+import org.ensime.model._
+import org.ensime.util.ensimefile._
+import org.ensime.util.file._
+import org.ensime.util.sourcefile._
+import org.ensime.vfs._
+import org.slf4j.LoggerFactory
+
 import scala.collection.immutable.{ Set => SCISet }
+import scala.collection.mutable
```
Comment (fommil) [inline core/src/main/scala/org/ensime/core/RichPresentationCompiler.scala]: more import weirdness

Comment (VC1995) [inline core/src/main/scala/org/ensime/core/RichPresentationCompiler.scala]: fixing real quick

**Comment by fommil** [comment]
one more question

**Comment by VC1995** [comment]
haha,yes?

**Pull request merged**